modules/compute/gke-node-pool/main.tf

```diff
@@ -67,9 +67,10 @@ resource "google_container_node_pool" "node_pool" {
   }
 
   dynamic "placement_policy" {
-    for_each = var.compact_placement ? [1] : []
+    for_each = var.placement_policy.type != null ? [1] : []
     content {
-      type = "COMPACT"
+      type        = var.placement_policy.type
```

Comment:
If the [only supported value](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_node_pool#placement_policy) of type is "COMPACT", do we need to take it from the user as an input? I'm wondering if we can pass it to the provider directly from this module